src/app/app.service.ts

```diff
@@ -2,11 +2,10 @@
 // Released under a MIT (SEI)-style license. See LICENSE.md in the project root for license information.
 
 import { Injectable } from '@angular/core';
-import { Http, Response, Headers, RequestOptions } from '@angular/http';
-import { Observable } from 'rxjs';
+import { HttpClient } from '@angular/common/http';
 
 @Injectable()
 export class AppService {
   // Resolve HTTP using the constructor
-  constructor(private _http: Http) {}
+  constructor(private _httpClient: HttpClient) {}
```

Comment:
@sei-aschlackman `@angular/http` is gone--I replaced it with this, does that make sense?

Reply:
yea, I can't believe we were still using Http!